webapp/components/test-runs-query.html

```diff
@@ -182,6 +187,13 @@
           if ('aligned' in params) {
             batchUpdate.aligned = params.aligned;
           }
+          if (this.masterRunsOnly) {
```

Comment:
I'm confused. Cica line 81 we model "master only" as a label, `master`. Here, we're modeling it as a property, `master:true`, and removing its corresponding label. Why don't we model it consistently one way or the other?

Reply:
Server-side URI params vs "less confusing UI" properties.